std_msgs/Header header
float64 electronic_voltage
float64 motor_voltage
float64 electronic_current
float64 motor_current